submissions/webdevtoday/oop_exercise/index.js

```diff
@@ -1,3 +1,75 @@
+<<<<<<< HEAD
```

Comment:
I think you don't need this file in this PR

Reply:
I've lost a part of work when I made commits( and I got this file. I don't understand how to fix it(